deployers/servicedeployer.go

```diff
@@ -119,7 +119,7 @@ func (deployer *ServiceDeployer) ConstructDeploymentPlan() error {
 
 	manifestReader.InitRootPackage(manifestParser, manifest)
 
-	if deployer.IsDefault == true && !utils.Flags.WithinOpenWhisk {
```

Comment:
@lionelvillard had added this flag to indicate that the wskdeploy is being run as an OpenWhisk action itself. If it is being run as an OpenWhisk action than skip this whole block of execution where wskdeploy is reading and inspecting project directory where action files are located, reading content of the action files and setting exec.Code of an ActionRecord. Hope this helps. We cannot set exec.Code this way if wskdeploy is running as an OpenWhisk action as none of the directory structure is accessible in cloud. But than I am trying to figure out myself what is the use of having wskdeploy run as an OpenWhisk action.